app.yml

```diff
@@ -40,7 +40,7 @@ default_events:
 - pull_request_review_comment
 - push
 # - release
-# - repository
+- repository
```

Comment:
Would it require a new permissions from users?

Reply:
Nope, we're not adding a new permission, just subscribing to a new event.